packages/cashier/src/components/percentage-selector/percentage-selector.tsx

```diff
@@ -3,7 +3,7 @@ import { Text } from '@deriv/components';
 import { formatMoney, getCurrencyDisplayCode, getDecimalPlaces } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
 import { TReactMouseEvent } from '../../types';
-import { useExchangeRate } from '@deriv/hooks';
+import { useExchangeRate2 } from '@deriv/hooks';
```

Comment:
Please leave a uniq comment above the codes need to be removed or replaced after QA checkin